### PR TITLE
[Storage]Release Note

### PR DESCRIPTION
**Pull request by xiafu-msft**

### CODE REVIEW

File: releases/2019-12/2019-12-python.md [hunk 1]
```diff
@@ -10,6 +10,16 @@ permalink: /releases/2019-12/python.html
 
 The Azure SDK team is pleased to make available the December 2019 client library preview release. This represents the first release of the ground-up rewrite of the client libraries to ensure consistency, idiomatic design, and excellent developer experience and productivity. This preview release includes new client libraries for Azure Cosmos, Identity, Key Vault (certificates, keys and secrets), Event Hubs, and Storage (blob, file-shares and queues).
 
+This release includes the following:
+
+#### GA
```
Comment (rakshith91) [inline releases/2019-12/2019-12-python.md]: Do we need to add anything about blobs and queues here?

File: releases/2019-12/python-quicklinks.md [hunk 2]
```diff
@@ -10,8 +10,8 @@
...
+| Storage - File DataLake | [azure-storage-file-datalake - 12.0.0b6](https://pypi.org/project/azure-storage-file-datalake/12.0.0b6) | [ReadMe](https://github.com/Azure/azure-sdk-for-python/blob/azure-storage-file-datalake_12.0.0b6/sdk/storage/azure-storage-file-datalake/README.md) | [Samples](https://github.com/Azure/azure-sdk-for-python/blob/azure-storage-file-datalake_12.0.0b6/sdk/storage/azure-storage-file-datalake/samples) | N/A | [ChangeLog](https://github.com/Azure/azure-sdk-for-python/blob/azure-storage-file-datalake_12.0.0b6/sdk/storage/azure-storage-file-datalake/HISTORY.md) |
```
Comment (rakshith91) [inline releases/2019-12/python-quicklinks.md]: No API ref doc for datalake?

File: releases/2019-12/2019-12-python.md (hunk 1 above)
Comment (xiafu-msft) [reply releases/2019-12/2019-12-python.md]: good question! any idea about how to add it?

File: releases/2019-12/python-quicklinks.md (hunk 2 above)
Comment (xiafu-msft) [reply releases/2019-12/python-quicklinks.md]: Good catch! there is!

Comment (adrianhall) [reply releases/2019-12/python-quicklinks.md]: These tables are auto-generated now - see @weshaggard for details.

File: releases/2019-12/2019-12-python.md [hunk 3]
```diff
@@ -35,6 +46,14 @@ If you have a bug or feature request for one of the libraries, please post an is
 
 Detailed change logs are linked to in the Quick Links below. Here are some critical call outs.
 
+### Storage
+
+- All the clients now have a `close()` method to close the sockets opened by the client when using without a context manager.
```
Comment (adrianhall) [inline releases/2019-12/2019-12-python.md]: using -> used

File: releases/2019-12/2019-12-python.md [hunk 4]
```diff
@@ -35,6 +46,14 @@ If you have a bug or feature request for one of the libraries, please post an is
...
+- Added `download_blob` method to the `ContainerClient`.
+- Added `delete_directory` method to the `ShareClient`.
```
Comment (adrianhall) [inline releases/2019-12/2019-12-python.md]: This isn't meant to be an exhaustive list of every single change - just things that are notable.  I don't think these two rise to that level.

File: releases/2019-12/2019-12-python.md (hunk 1 above)
Comment (xiafu-msft) [reply releases/2019-12/2019-12-python.md]: Checked with @zezha-msft since we have announced blob and queue was GA, we don't need to add them this time

File: releases/2019-12/python-quicklinks.md (hunk 2 above)
Comment (weshaggard) [reply releases/2019-12/python-quicklinks.md]: Yes undo the changes the quick links. Verify the information for your package in https://github.com/Azure/azure-sdk/pull/837.

File: releases/2019-12/2019-12-python.md [hunk 5]
```diff
@@ -43,6 +50,12 @@ If you have a bug or feature request for one of the libraries, please post an is
 
 Detailed change logs are linked to in the Quick Links below. Here are some critical call outs.
 
+### Storage
+
+- All the clients now have a `close()` method to close the sockets opened by the client when used without a context manager.
+- In FileShare`close_handle(handle)` and `close_all_handles()` no longer return int. These functions return a `dict(str, int)` which has 
+the number of handles closed. The dictionary will be like `{"closed_handles_count": 6}`.
+
```
Comment (adrianhall) [inline releases/2019-12/2019-12-python.md]: These items don't rise to the "key items" bar.  We basically fixed a few bugs - not news.  This is good for a CHANGELOG - not for release notes.

Comment (xiafu-msft) [reply releases/2019-12/2019-12-python.md]: They are not bugs I think

File: releases/2019-12/python-quicklinks.md [hunk 6]
```diff
@@ -10,8 +10,8 @@
...
-| Storage - Blobs | [azure-storage-blob - 12.0.0](https://pypi.org/project/azure-storage-blob/12.0.0) | [ReadMe](https://github.com/Azure/azure-sdk-for-python/blob/azure-storage-blob_12.0.0/sdk/storage/azure-storage-blob/README.md) | [Samples](https://github.com/Azure/azure-sdk-for-python/blob/azure-storage-blob_12.0.0/sdk/storage/azure-storage-blob/samples) | [Api Reference](https://azuresdkdocs.blob.core.windows.net/$web/python/azure-storage-blob/12.0.0/index.html) | [ChangeLog](https://github.com/Azure/azure-sdk-for-python/blob/azure-storage-blob_12.0.0/sdk/storage/azure-storage-blob/HISTORY.md) |
```
Comment (weshaggard) [inline releases/2019-12/python-quicklinks.md]: This file no longer exists so undo these changes.

**Comment by weshaggard** [comment]
@xiafu-msft what is the status on this PR?

**Comment by adrianhall** [comment]
I'm going to add the required entries to the python release notes in a separate PR, then close this one.

**Comment by adrianhall** [comment]
Replaced by #878
